docs/get-details/dapps/smart-contracts/apps/index.md

```diff
@@ -406,8 +406,7 @@ With the release of TEAL 6 (AVM 1.1), Smart Contracts may issue inner transactio
 
     - An application may not call itself, even indirectly. This is referred to as `re-entrancy` and is explicitly forbidden. 
     - An application may only call into other applications up to a stack depth of 8. In other words if app calls (`->`) look like 1->2->3->4->5->6->7->8, App 8 may _not_ call another application. This would violate the stack depth limit.
-    - An application may issue up to 256 inner transactions to increase its budget (max budget of 179.2k!), but the max call budget is shared for all applications in the group. Meaning you cant have two apps calls in the same group that _both_ try to issue 256 inner app calls.
-    - An application may _not_ call contracts with an older program version. This limitation prevents an older application from unexpected behavior introduced in newer program versions 
+    - An application may issue up to 256 inner transactions to increase its budget (max budget of 179.2k!), but the max call budget is shared for all applications in the group. Meaning you cant have two apps calls in the same group that _both_ try to issue 256 inner app calls. 
```

Comment:
```suggestion
    - An application may issue up to 256 inner transactions to increase its budget (max budget of 179.2k!), but the max call budget is shared for all applications in the group. Meaning you cant have two apps calls in the same group that _both_ try to issue 256 inner app calls. 
    - An application may _not_ call contracts with a program version of 3 or below. This limitation prevents an older application from unexpected behaviour introduced in newer program versions.
```

Reply:
I believe there is still a limitation of sorts. You can only call inner transactions to program version 4 or above. You still can't call a `#pragma version 3` program or below via an inner transaction.

Reply:
yes, that's true. I don't know what level of faithfulness you want to include in the docs so I wasn't sure whether you'd require we include that information (which is more complicated to explain unfortunately). I wanted to remove the way it was currently worded because it caused some people to panic